tracer/src/Datadog.Trace/Logging/DirectSubmission/DirectLogSubmissionSettings.cs

```diff
@@ -161,8 +206,8 @@ public DirectLogSubmissionSettings(IConfigurationSource? source, IConfigurationT
         internal string? ApiKey { get; }
 
         /// <summary>
-        /// Gets or sets whether logs injection has been explicitly enabled or disabled
+        /// Gets or sets a value indicating whether logs injection has been explicitly enabled or disabled
```

Comment:
I wonder if there's anywhere else that conditions on whether it's _explicitly_ enabled or disabled 🤔 

```suggestion
        /// Gets or sets a value indicating whether logs injection is enabled or disabled
```

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/5126/commits/7d4b6a84d089ba85674e50047ce60bc18a403cfc